### PR TITLE
pigz: Update to 2.6

### PR DESCRIPTION
**Pull request by ligaard**
#### Description

Update to version 2.6.

###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on
macOS 11.5 20G71 arm64
Xcode 12.5.1 12E507

and

macOS 11.5 20G71 x86_64
Xcode 12.5.1 12E507

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] checked your Portfile with `port lint`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

File: archivers/pigz/Portfile [hunk 1]
```diff
@@ -19,13 +19,12 @@ long_description    pigz, which stands for parallel implementation of gzip, \
...
 
-checksums           rmd160  99ffd6ca6acbf29c436d80e814a144dfd1bd3337 \
-                    sha256  a4f816222a7b4269bd232680590b579ccc72591f1bb5adafcd7208ca77e14f73
+checksums           rmd160  42bd149f6a639d7db8517c23f3f0ccf6b0d7524a \
+                    sha256  2eed7b0d7449d1d70903f2a62cd6005d262eb3a8c9e98687bc8cbb5809db2a7d \
+                    size    106840
 
 depends_lib         port:zlib
 
-patchfiles          patch-Makefile.diff
```
Comment (danchr) [inline archivers/pigz/Portfile:27]: If you look at the build log, it now uses `gcc`, indicating that this change — or something similar — remains required.

**Review by danchr — CHANGES_REQUESTED** [review]

File: archivers/pigz/Portfile (hunk 1 above)
Comment (ligaard) [reply archivers/pigz/Portfile]: Updated the patch file and can see it now uses clang.

**Comment by danchr** [comment]
This looks good overall, but do note that MacPorts requires linear history and has a policy on commit messages, as pointed out in the description. Could you rebase and squash this into a single commit, without merges, and adjust the message? Thanks!

**Comment by ligaard** [comment]
I don't know how to do this from the GitHub interface and I cannot find a way to checkout this pull request (I see GitHub kindly offers me to install their software, but I am not inclined to go further down that path; I expected this to be a 30 min. "be kind" operation and simply do not have another minute to spare on this now. Sorry).

**Comment by danchr** [comment]
> I don't know how to squash from the GitHub interface and I cannot find a way to checkout this pull request (I see GitHub kindly offers me to install their software, but I am not inclined to go further down that path; I expected this to be a 30 min. "be kind" operation and simply do not have another minute to spare on this now. Sorry).

Fair enough; I'll do it when I'm in front of a computer. Thank you for your contribution!

**Comment by danchr** [comment]
Manually rebased and applied as 1788d8e8a019764f69970821896de39024f13b6c; thanks again!

**Comment by ligaard** [comment]
Thanks a bunch!

I know this is now a support channel but how did you do it? I mean: The commits were pushed already, so what are the tricks to know..?

**Comment by danchr** [comment]
> Thanks a bunch!
> 
> I know this is not a support channel but how did you do it? I mean: The commits were pushed already, so what are the tricks to know..?

Erm, the _really_ short version is that I used Git, rather than the Github's web UI — see [the Guide](https://guide.macports.org/#project.github) for details. If using Git is too complicated for you, you might want to just report tickets on [Trac](http://trac.macports.org) 🙂

**Pull request closed without merging**